apidoc/Titanium/UI/View.yml

```diff
@@ -1063,6 +1063,23 @@ properties:
     default: 0
 
     
+  - name: touchFeedback
+    summary: A material design visual construct that provides an instantaneous visual confirmation of touch point.
+    description: |
+        This is an opt-in feature available from Android Lollipop. 
+        Touch feedback is applied only if the backgroundColor is a solid color.
+    type: Boolean
+    default: false
+    platforms: [android]
+
+    
+  - name: touchFeedbackColor
+    summary: Optional touch feedback ripple color. 
```

Comment:
You should mention that this property does nothing unless `touchFeedback` is true